.env

```diff
@@ -0,0 +1,6 @@
+# The URL of our Auth0 Tenant Domain.
+# If you're using a Custom Domain, be sure to set this to that value instead.
+AUTH0_DOMAIN='https://dev-i4mvq7is6cozbf5d.us.auth0.com/'
+
+# Our Auth0 API's Identifier.
+AUTH0_AUDIENCE='http://controllers.juicelabs.com'
```

Comment:
Should we be adding this to the repo? Shouldn't it be a file that is created on the controller server when launching the controller?

Reply:
the audience is like a namespace for JWT tokens - it's an arbitrary string that's a URL by convention. In this case the audience identifies that we want to connect to "controllers" using this token.